src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothServerSocket.cs

```diff
@@ -62,17 +100,14 @@ private void RegisterAcceptStateChangedEvent()
             _connectionStateChangedCallback = (int result, BluetoothSocketState connectionState, ref SocketConnectionStruct socketConnection, IntPtr userData) =>
             {
                 Log.Info(Globals.LogTag, "AcceptStateChanged cb is called");
-                if (_acceptStateChanged != null)
-                {
-                    BluetoothSocket socket = new BluetoothSocket();
-                    socket.connectedSocket = socketConnection.SocketFd;
-                    GCHandle handle2 = (GCHandle) userData;
-                    _acceptStateChanged(handle2.Target as BluetoothServerSocket, new AcceptStateChangedEventArgs((BluetoothError)result, connectionState, BluetoothUtils.ConvertStructToSocketConnection(socketConnection), socket));
-                }
+                BluetoothSocket socket = new BluetoothSocket();
+                socket.connectedSocket = socketConnection.SocketFd;
+                socket.remoteAddress = socketConnection.Address;
+                socket.serviceUuid = socketConnection.ServiceUuid;
+                _acceptStateChanged?.Invoke(null, new AcceptStateChangedEventArgs((BluetoothError)result, connectionState, BluetoothUtils.ConvertStructToSocketConnection(socketConnection), socket));
```

Comment:
new AcceptStateChangedEventArgs((BluetoothError)result, connectionState, BluetoothUtils.ConvertStructToSocketConnection(socketConnection), socket)

``` C#
internal AcceptStateChangedEventArgs(BluetoothError result, BluetoothSocketState state, SocketConnection connection, BluetoothSocket server)
```
Let see, The 5th parameter is server but, socket is not ServerSocket

Why sender is null?

Reply:
AcceptStateChangedEventArgs() delivers IBluetoothServerSocket containing SocketFd of connected client to send data to client.